docs/install/install-methods.md

```diff
@@ -1,17 +1,90 @@
 ---
 id: install-methods
-title: Installation Methods
+title: Installation
 sidebar_position: 2
 ---
 
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
 # Installation Methods
 
-Vignette provides a lot of official ways for you to get the software. We have listed what we support so far below, however, if you don't see your
-target platform, please [raise an issue](https://github.com/vignetteapp/vignette/issues/new) for your platform and see if we can support it.
+Vignette provides a lot of official ways for you to get the software. We have listed what we support so far below, however, if you don't see your target platform, please [raise an issue](https://github.com/vignetteapp/vignette/issues/new) for your platform and see if we can support it.
+
+<Tabs groupId="operating-systems">
+  <TabItem value="microsoft-store" label="Microsoft Store (Windows)" default>
+  
+  Vignette is Coming Soon to the Microsoft Store! 
+  
+  
+  Until then, please consinder installing it via our Portable Installation Guide
+  </TabItem>
+  <TabItem value="steam" label="Steam">
+  Vignette is Coming Soon to Steam!
+  
+
+  Until then, please consinder installing it via our Portable Installation Guide or AUR Installation Guide.
```

Comment:
Maybe remove this new line to match with Windows Store tab to be consistent? or vice versa
![image](https://user-images.githubusercontent.com/34704796/135729904-a1b6936d-3d9f-492c-99fa-2cc366887628.png)
![image](https://user-images.githubusercontent.com/34704796/135729909-aeb54564-7935-4192-8ab2-7a64707c3769.png)

Reply:
Wanted to add it to the Microsoft Store one too,  but it didn't actually put the newline there.

Reply:
Hmm, maybe try adding double spaces at the end.
Ex: `Vignette is Coming Soon to the Microsoft Store!  Until then, ...`